From 1890c90f7c47cf2fa2d46dd5f6b8654619f25b6c Mon Sep 17 00:00:00 2001
From: Siva Mahadevan <me@svmhdvn.name>
Date: Sat, 26 Feb 2022 18:30:59 -0500
Subject: [PATCH 04/12] target: correctly set CPU_CFLAGS_74kc for MIPS 74kc

---
 include/target.mk | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/include/target.mk b/include/target.mk
index 7526224972..ff2fbaf291 100644
--- a/include/target.mk
+++ b/include/target.mk
@@ -191,20 +191,20 @@ LINUX_RECONF_DIFF = $(SCRIPT_DIR)/kconfig.pl - '>' $(call __linux_confcmd,$(filt
 ifeq ($(DUMP),1)
   BuildTarget=$(BuildTargets/DumpCurrent)
 
-  CPU_CFLAGS = -Os -pipe
+  CPU_CFLAGS = -O2 -pipe
   ifneq ($(findstring mips,$(ARCH)),)
     ifneq ($(findstring mips64,$(ARCH)),)
       CPU_TYPE ?= mips64
     else
       CPU_TYPE ?= mips32
     endif
     CPU_CFLAGS += -mno-branch-likely
     CPU_CFLAGS_mips32 = -mips32 -mtune=mips32
     CPU_CFLAGS_mips64 = -mips64 -mtune=mips64 -mabi=64
     CPU_CFLAGS_mips64r2 = -mips64r2 -mtune=mips64r2 -mabi=64
     CPU_CFLAGS_4kec = -mips32r2 -mtune=4kec
     CPU_CFLAGS_24kc = -mips32r2 -mtune=24kc
-    CPU_CFLAGS_74kc = -mips32r2 -mtune=74kc
+    CPU_CFLAGS_74kc = -march=74kc -mtune=74kc -mno-mips16 -mabi=32 -msym32 -mno-interlink-compressed -mframe-header-opt -Wa,-march=74kc,-mtune=74kc,-mno-mips16,-mabi=32,-msym32
     CPU_CFLAGS_octeonplus = -march=octeon+ -mabi=64
   endif
   ifeq ($(ARCH),i386)
@@ -290,9 +290,6 @@ ifeq ($(DUMP),1)
     ifneq ($(CONFIG_VIRTIO),)
       FEATURES += virtio
     endif
-    ifneq ($(CONFIG_CPU_MIPS32_R2),)
-      FEATURES += mips16
-    endif
     FEATURES += $(foreach v,6 7,$(if $(CONFIG_CPU_V$(v)),arm_v$(v)))
 
     # remove duplicates
-- 
2.25.1

