From 1e7e0cd0acdd362ebd95ae965173d43c8b80e818 Mon Sep 17 00:00:00 2001
From: Siva Mahadevan <me@svmhdvn.name>
Date: Sun, 9 Jul 2023 17:19:15 -0400
Subject: [PATCH] toolchain/gcc: improve CONFIGURE options

---
 toolchain/gcc/final/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/toolchain/gcc/final/Makefile b/toolchain/gcc/final/Makefile
index 78a5576b..7fc95727 100644
--- a/toolchain/gcc/final/Makefile
+++ b/toolchain/gcc/final/Makefile
@@ -7,6 +7,8 @@ GCC_CONFIGURE += \
 	--enable-languages=$(TARGET_LANGUAGES) \
 	--enable-shared \
 	--enable-threads \
+	--with-mips-plt \
+	--disable-nls \
 	--with-slibdir=$(TOOLCHAIN_DIR)/lib \
 	--enable-plugins \
 	--enable-lto \
-- 
2.41.0

