From 2c8fd36979e346a50b0382aec557faaaa6e3d348 Mon Sep 17 00:00:00 2001
From: Siva Mahadevan <me@svmhdvn.name>
Date: Sat, 26 Feb 2022 18:34:25 -0500
Subject: [PATCH 10/12] toolchain: disable USE_MIPS16

---
 toolchain/Config.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/toolchain/Config.in b/toolchain/Config.in
index 366f5c8b48..5be18b0529 100644
--- a/toolchain/Config.in
+++ b/toolchain/Config.in
@@ -28,7 +28,7 @@ menuconfig TARGET_OPTIONS
 	config USE_MIPS16
 		bool "Build packages with MIPS16 instructions" if TARGET_OPTIONS
 		depends on HAS_MIPS16
-		default y
+		default n
 		help
 		  If your target CPU does support the MIPS16 instruction set
 		  and you want to use it for packages, enable this option.
-- 
2.25.1

