From 45dd3e268bcfb53ce8c2b76443de5eec6ed64858 Mon Sep 17 00:00:00 2001
From: Siva Mahadevan <me@svmhdvn.name>
Date: Sat, 26 Feb 2022 18:33:53 -0500
Subject: [PATCH 06/12] ath79: set default CPU_TYPE to 74kc

---
 target/linux/ath79/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/linux/ath79/Makefile b/target/linux/ath79/Makefile
index 2a9857bfb1..b912937ee7 100644
--- a/target/linux/ath79/Makefile
+++ b/target/linux/ath79/Makefile
@@ -3,7 +3,7 @@ include $(TOPDIR)/rules.mk
 ARCH:=mips
 BOARD:=ath79
 BOARDNAME:=Atheros ATH79
-CPU_TYPE:=24kc
+CPU_TYPE:=74kc
 SUBTARGETS:=generic mikrotik nand tiny
 
 FEATURES:=ramdisk squashfs usbgadget
-- 
2.25.1

